templates/html/.partials/operation.html

```diff
@@ -4,36 +4,41 @@
 
 {% macro operation(operation, operationType, channelName, channel) %}
 
-<a name="operation-{{operationType}}-{{channelName}}"></a>
-<div class="center-block p-8">
-  <div class="operation pt-8 pb-8">
-    <h3 class="font-mono text-base">
-      {% if operationType === 'publish' %}
-      <span class="font-mono border border-blue text-blue uppercase p-1 rounded" title="Publish">Pub</span>
-      {% endif %}
-      {% if operationType === 'subscribe' %}
-      <span class="font-mono border border-green-dark text-green-dark uppercase p-1 rounded" title="Subscribe">Sub</span>
-      {% endif %}
-      {{ slicedString(channelName) }}
-    </h3>
-
-    <div class="mt-4 mb-4 markdown">{{ channel.description() | markdown2html | safe }}</div>
-
-    <p class="text-grey text-sm">{{operation.summary()}}</p>
-    <div class="mt-4 mb-4 markdown">{{ operation.description() | markdown2html | safe }}</div>
-
-    {% if operation.hasMultipleMessages() %}
-      <p>Accepts <strong>one of</strong> the following messages:</p>
-      {% for msg in operation.messages() %}
-        {{ message(msg, showIndex=true, index=loop.index, open=false) }}
-      {% endfor %}
-    {% else %}
-      <p>Accepts the following message:</p>
-      {{ message(operation.message(0), showIndex=false, open=true) }}
-    {% endif %}
-
-    {{ tags(operation.tags()) }}
+<div class="responsive-container">
+  <a name="operation-{{operationType}}-{{channelName}}"></a>
+  <div class="center-block p-8">
+    <div class="operation pt-8 pb-8">
+      <h3 class="font-mono text-base">
+        {% if operationType === 'publish' %}
+        <span class="font-mono border border-blue text-blue uppercase p-1 rounded" title="Publish">Pub</span>
+        {% endif %}
+        {% if operationType === 'subscribe' %}
+        <span class="font-mono border border-green-dark text-green-dark uppercase p-1 rounded" title="Subscribe">Sub</span>
+        {% endif %}
+        {{ slicedString(channelName) }}
+      </h3>
+
+      <div class="mt-4 mb-4 markdown">{{ channel.description() | markdown2html | safe }}</div>
+
+      <p class="text-grey text-sm">{{operation.summary()}}</p>
+      <div class="mt-4 mb-4 markdown">{{ operation.description() | markdown2html | safe }}</div>
+
+
+      {{ tags(operation.tags()) }}
+    </div>
   </div>
 </div>
 
+{% if operation.hasMultipleMessages() %}
+  {% for msg in operation.messages() %}
+    <div class="responsive-container message-container">
+      {{ message(msg, showIndex=true, index=loop.index, open=false, prefixHtml='<p>Accepts <strong>one of</strong> the following messages:</p>') }}
```

Comment:
This prefixHtml seems out of place. It's not the message who accepts **one of** the messages, it's the operation, therefore I think this logic shouldn't be delegated to the message template.

Also, seems a bit hacky to pass HTML as a param, don't you think so? I appreciate the effort you did here to DRY things but maybe making it a little bit less generic will work better. Why don't you simply put this HTML before calling the `message` template?